utils/python/CIME/utils.py

```diff
@@ -8,7 +8,7 @@
 TESTS_FAILED_ERR_CODE = 100
 logger = logging.getLogger(__name__)
 
-def expect(condition, error_msg, exc_type=SystemExit):
+def expect(condition, error_msg, exc_type=SystemExit, error_prefix="ERROR: "):
```

Comment:
@jedwards4b this default value has a whitespace after the colon but then another whitespace is added later, this is causing some unit tests to fail since the exception string is slighting different. I'm going to hotfix this.